components/dashboard/src/settings/Integrations.tsx

```diff
@@ -158,6 +158,7 @@ function GitProviders() {
             await openAuthorizeWindow({
                 host,
                 scopes,
+                overrideScopes: true,
```

Comment:
I think this is the single place where we actually want to override the scopes.
In all other cases we want to elevate scopes anyways.